lib/ansible/modules/cloud/vmware/vmware_host_kernel_manager.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2019, Aaron Longchamps, <a.j.longchamps@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_host_kernel_manager
+short_description: Configure kernel module options on an ESXi host or cluster.
+description:
+>-
```

Comment:
Don't do this, make these items in your description-list.

Reply:
Comment has been adjusted.